var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -150,9 +150,9 @@ class PyNumpy(PythonPackage):
     # NVHPC support added in https://github.com/numpy/numpy/pull/17344
     conflicts("%nvhpc", when="@:1.19")
 
-    # Newer versions will not build with Intel https://github.com/numpy/numpy/issues/22011
-    conflicts("%intel", when="@1.23.0:")
-    conflicts("%oneapi", when="@1.23.0:")
+    # See https://github.com/numpy/numpy/issues/22011
+    conflicts("%intel", when="@1.23.0:1.23.3")
+    conflicts("%oneapi", when="@1.23.0:1.23.3")
```

Comment:
This PR does not change anything to make oneapi work. Do you know what the problem was ? PKG_CONFIG_PATH is set in intel-oneapi-mkl/package.py, but the gitlab tests were failing and I could not track it down.

Reply:
Yes, that has nothing to do with the rest of this PR. That particular issue was a C++ not finding a few heapsort function calls. numpy changed them to be explicitly exported in a separate header file, so it works with intel&oneapi now.